### DISCUSSION TITLE
issue, epic, sprint context

### DISCUSSION THREAD

**Discussion by stchar** [ankitpokhrel/jira-cli#302] [Ideas]
Let's say I have an Issue I'm working on.  It would be cool if I can put the issue key in the some cache. 
like `jira issue active SRV-12345` so I don't need to remember it.

Once I have this context  all commands like `jira issue comment add` `jira issue move` would address that key by default.  

I believe there is a workaround - bash alias, but would be nice to have it implemented. 
```
export ISSUE=SRV-12345
export EPIC=EP-67890
alias issue-comment='jira issue comment add  $ISSUE '
alias epic-link='jira epic link $EPIC '
```

**Comment by ankitpokhrel** [comment]
That's another level of laziness 👀
